static/js/hotkey.js

```diff
@@ -450,6 +450,11 @@ export function process_enter_key(e) {
         return true;
     }
 
+    // If we got this far and the modal is open, then avoid the enter press.
+    if (overlays.is_modal_open()) {
+        return false;
+    }
```

Comment:
I think this may be clearer done a few blocks up: Just below the "button role" block.